packages/two_dimensional_scrollables/test/table_view/table_test.dart

```diff
@@ -1040,6 +1040,59 @@ void main() {
       );
     });
 
+    testWidgets('paint rects are correct when reversed and pinned',
+        (WidgetTester tester) async {
+      // TODO(Piinks): Rewrite this to remove golden files from this repo when
+      //  mock_canvas is public - https://github.com/flutter/flutter/pull/131631
+      // foreground, background, and precedence per mainAxis
+      final TableView tableView = TableView.builder(
+        verticalDetails: const ScrollableDetails.vertical(reverse: true),
+        horizontalDetails: const ScrollableDetails.horizontal(reverse: true),
+        rowCount: 2,
+        pinnedRowCount: 1,
+        columnCount: 2,
+        pinnedColumnCount: 1,
+        columnBuilder: (int index) => TableSpan(
+          extent: const FixedTableSpanExtent(200.0),
+          foregroundDecoration: const TableSpanDecoration(
+              border: TableSpanBorder(
+                  trailing: BorderSide(
+            color: Colors.orange,
+            width: 3,
+          ))),
```

Comment:
formatting nit (probably just missing some trailing commas?):

```suggestion
          foregroundDecoration: const TableSpanDecoration(
            border: TableSpanBorder(
              trailing: BorderSide(
                color: Colors.orange,
                width: 3,
              ),
            ),
          ),
```

Reply:
Good catch, I found a couple more cases of this, fixed them all. :)